app/controllers/time_regs_controller.rb

```diff
@@ -27,6 +28,8 @@ def new_modal
   def create
     @time_reg = current_user.time_regs.new(time_reg_params.except(:project_id, :minutes_string))
 
+    @time_regs_week.find(&:active?)&.deactivate! if @time_reg.valid?
```

Comment:
Should we deactivate it for them, without them knowing? I think it would be simple to throw back the validation from the model and let them choose what to do with the active `time_reg`. And we will be having a banner in the view that on clicking redirects to the active time_reg.

Reply:
Yes, of course. That sounds much simpler.

I was trying to replicate the functionality in the software we already use: 
When I have a timer running, and start a different timer, the first timer stops and the other starts.
And since we only allow timers to be started on the current day, the user should be able to clearly see the change.

I can see my logic is wrong too.

Reply:
I understand I didn't know, Would a confirmation dialog be informative? That you have another running time_registration that will be stopped. 

I'm just thinking of ways to make sure everyone knows and understands they're stopping another existing running timer.

Reply:
Displaying a confirmation dialog when starting another timer sounds nice.
How can we do it when starting a timer from the "new" modal? Should we open the dialog from that modal, or maybe show some kind of warning text in the form?

Reply:
I think in this PR, we can keep it simple first and inform the user that there's another `time_reg`. Clicking on the top banner would take them to the active `time_reg` and they'll resolve the issue.

Then following this one(later on):
1. We can choose to let them know within that time_reg modal using a `PopConfirm` component, something like https://ant.design/components/popconfirm
2. Give thoughts on implementing a daily cleanup job.